crates/rolldown_error/src/events/ambiguous_external_namespace.rs

```diff
@@ -4,14 +4,19 @@ use oxc::span::Span;
 
 use crate::{diagnostic::Diagnostic, types::diagnostic_options::DiagnosticOptions, EventKind};
 
+#[derive(Debug)]
+pub struct Namespace {
```

Comment:
The naming here is quite confused. I can't think of a better one now. But let's not use `Namespace` such a common name.

```suggestion
pub struct AmbiguousExternalNamespaceModule {
```